docs/root/intro/arch_overview/http_routing.rst

```diff
@@ -87,6 +88,16 @@ headers <config_http_filters_router_headers_consumed>`. The following configurat
 Note that retries may be disabled depending on the contents of the :ref:`x-envoy-overloaded
 <config_http_filters_router_x-envoy-overloaded_consumed>`.
 
+.. _arch_overview_http_routing_hedging:
+
+Request Hedging
```

Comment:
Can you add some more detail on how this interacts with retries etc.?

Reply:
+1, also can you be more explicit on how hedging on per_try_timeout effectively disables the per_try_timeout and let's the original request continue on? That wasn't clear to me until I started looking through the implementation.

Reply:
updated the wording here, I think it's more clear now but let me know if it can use more work